app/src/pages/inside/dashboardItemPage/modals/widgetWizardModal/widgetWizardContent/widgetWizardContent.jsx

```diff
@@ -166,7 +166,11 @@ export class WidgetWizardContent extends Component {
   };
 
   onClickPrevStep = () => {
-    this.setState({ step: this.state.step - 1 });
+    const activeWidget = this.widgets.find(
+      (widget) => widget.id === this.props.formValues.widgetType,
+    );
+    const skipSecondStep = !activeWidget?.controls;
+    this.setState({ step: this.state.step - (skipSecondStep ? 2 : 1) });
```

Comment:
Let's move this to a dedicated function, e.g. `getStepIncrement` to use for `nextStep` as well.